harvester/metadata.py

```diff
@@ -222,18 +205,83 @@ def get_html_content_metadata(cls, html_content: str | bytes) -> dict:
-            og_tag = soup.find("meta", property=og_tag_name)
+        tags.update(cls._parse_open_graph_meta_elements(html_soup))
+        tags.update(cls._parse_dublin_core_meta_elements(html_soup))
+
+        return tags
+
+    @classmethod
+    def _parse_open_graph_meta_elements(cls, html_soup: BeautifulSoup) -> dict:
+        """Parse OpenGraph <meta> tag values.
+
+        https://ogp.me/
+
+        Example:
+            <meta property="og:title" content="MIT Libraries"/>
+            <meta property="og:description" content="Library home page." />
+        """
+        og_tag_properties = (
+            "og:site_name",
+            "og:title",
+            "og:locale",
+            "og:type",
+            "og:image",
+            "og:url",
+            "og:image:url",
+            "og:image:secure_url",
+            "og:image:type",
+            "og:image:width",
+            "og:image:height",
+            "og:image:alt",
+            "og:description",
+        )
+
+        fields = {}
+        for og_tag_property in og_tag_properties:
+            og_tag = html_soup.find("meta", attrs={"property": og_tag_property})
             if og_tag is None:
                 continue
             if content := og_tag.get("content"):  # type: ignore[union-attr]
                 content_stripped = content.strip()  # type: ignore[union-attr]
                 if content_stripped != "":
-                    og_tag_name_friendly = og_tag_name.replace(":", "_")
-                    tags[og_tag_name_friendly] = content_stripped
-        return tags
+                    og_tag_name_friendly = og_tag_property.replace(":", "_")
+                    fields[og_tag_name_friendly] = content_stripped
+        return fields
+
+    @classmethod
+    def _parse_dublin_core_meta_elements(cls, html_soup: BeautifulSoup) -> dict:
+        """Parse Dublin Core <meta> tag values.
+
+        Example:
+            <meta name="DC.Title" content="LibGuides: Biology: Home"/>
+            <meta name="DC.Description" content="A guide about biology."/>
+        """
+        dc_tag_names = (
+            "DC.Title",
+            "DC.Creator",
+            "DC.Subject",
+            "DC.Description",
+            "DC.Publishers",
+            "DC.Rights",
+            "DC.Language",
+            "DC.Identifier",
+            "DC.Date.Created",
+            "DC.Date.Modified",
+        )
+        fields = {}
+        for dc_tag_name in dc_tag_names:
+            dc_tag = html_soup.find("meta", attrs={"name": dc_tag_name})
+            if dc_tag is None:
+                continue
+            if content := dc_tag.get("content"):  # type: ignore[union-attr]
+                content_stripped = content.strip()  # type: ignore[union-attr]
+                if content_stripped != "":
+                    dc_tag_name_friendly = dc_tag_name.replace(":", "_")
+                    fields[dc_tag_name_friendly] = content_stripped
+        return fields
```

Comment:
Optional: since there's a lot of repetition between `_parse_open_graph_meta_elements` and `_parse_dublin_core_meta_elements`, you could consider a single method that adds a tag list option and has an `og` and `dc` option. But's that admittedly a more complicated signature so that's why it's optional!